cdf-fabric-replicator-chart/values.yaml

```diff
@@ -0,0 +1,76 @@
+# Default values for cdf-fabric-replicator-chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository:
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "1.0.0"
+
+replicaCount: 1
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+podAnnotations: {}
+
+podSecurityContext:
+  runAsUser: 1000
+  runAsGroup: 3000
+  fsGroup: 2000
+
+securityContext:
+  runAsNonRoot: true
+  readOnlyRootFilesystem: true
```

Comment:
Getting file permission errors as a result of adding these securityContext settings

Reply:
I tested this, the issue occurs with the dockerfile and the entrypoint.sh, we already have a fix but it is in the slim docker image. I am going to remove the security context in this pr and add it to the pr with the slim docker image

Reply:
Made a new PR to introduce this change https://github.com/cognitedata/cdf-fabric-replicator/pull/92